.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,29 +1,24 @@
 name: "Bug Report"
-description: Report a bug with Kurtosis
+description: Report broken functionality
 labels: ["bug"]
 body:
-  - type: markdown
-    attributes:
-      value: |
-        Thank you for taking the time to fill out this Bug Report. We value your feedback.
-        Please make sure to update the title of this Bug Report to concisely describe the issue.
   - type: input
     id: version
     attributes:
-      label: What version of the Kurtosis CLI were you running when you encountered this bug?
+      label: What version of the Kurtosis CLI were you running when you hit this bug?
       placeholder: e.g. 1.2.0
       description: |
-        To view the current version of the Kurtosis CLI you are using, run `kurtosis version`.
+        View the version of your Kurtosis CLI with `kurtosis version`.
```

Comment:
@mieubrisse do we want to request engine version too?

Reply:
For posterity: probably yes, but would require a code change to `kurtosis feedback`. Going to add a checkbox instead, for users to verify they've checked this